crates/cdk-sql-common/src/mint/mod.rs

```diff
@@ -1698,6 +1736,19 @@ where
                     .bind("signed_time", current_time as i64)
                     .execute(&self.inner)
                     .await?;
+
+                    query(
+                        r#"
+                        INSERT INTO keyset_amounts (keyset_id, type, amount)
+                        VALUES (:keyset_id, 'issued', :amount)
+                        ON CONFLICT (keyset_id, type)
+                        DO UPDATE SET amount = keyset_amounts.amount + :amount
```

Comment:
I wonder if it would be better if create the first entry with a zero amount and then update that each time instead of doing the insert with the on conflict.